database/migrations/2020_02_17_170000_create_markets_table.php.stub

```diff
@@ -35,13 +26,13 @@ class CreateMarketsTable extends Migration
             $table->text('faq_content')->nullable(); // Frequently Asked Questions about the market (such as where to park at each location in the market). Written in Markdown.
             $table->text('competitors_content')->nullable(); // Nice paragraph about the other escape rooms in each market. First used on /escape-rooms page. Written in Markdown.
 
-            $table->foreignId('image_id')->nullable()->references('id')->on($imageTable); // Cover image for market
-            $table->foreignId('ogimage_id')->nullable()->references('id')->on($imageTable); // External open graph image id. Featured image for social sharing. Will default to image_id unless this is used.
-            $table->foreignId('map_image_id')->nullable()->references('id')->on($imageTable); // Image of location map for the market.
-            $table->foreignId('video_id')->nullable()->references('id')->on($videoTable); // Featured video for the market.
+            $table->foreignId('image_id')->nullable()->references('id')->on(app('image')->getTable()); // Cover image for market
+            $table->foreignId('ogimage_id')->nullable()->references('id')->on(app('image')->getTable()); // External open graph image id. Featured image for social sharing. Will default to image_id unless this is used.
+            $table->foreignId('map_image_id')->nullable()->references('id')->on(app('image')->getTable()); // Image of location map for the market.
+            $table->foreignId('video_id')->nullable()->references('id')->on(app('video')->getTable()); // Featured video for the market.
 
-            $table->foreignId('creator_id')->references('id')->on($userTable);
-            $table->foreignId('updater_id')->references('id')->on($userTable);
+            $table->foreignId('creator_id')->references('id')->on(app('user')->getTable());
+            $table->foreignId('updater_id')->references('id')->on(app('user')->getTable());
```

Comment:
Let's use the new function since Laravel 8.14 of `$table->foreignIdFor()` as used in the tipoff/discounts package:

`$table->foreignIdFor(app('user'), 'creator_id');`

Reply:
Sure. Fixed in my latest commit.

Reply:
Thanks!